app/views/container/show.html.haml

```diff
@@ -0,0 +1,10 @@
+- if %w(container_routes container_services container_replicators container_groups container_nodes container_images container_templates).include?(@display)
+  = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@record.id}"}
+- elsif @showtype == "timeline"
+  = render :partial => "layouts/tl_show_async"
+- elsif @showtype == "performance"
+  = render :partial => "layouts/performance_async"
+- elsif @showtype == "main"
+  = render :partial => "layouts/textual_groups_generic"
+- elsif @showtype == "topology"
+  = render :file => 'container_topology/show'
```

Comment:
I dont think topology is relevant here. 

@himdel it seems this part is mostly the same for a lot of entities. any smart refactoring we can do for this?